pkgs/by-name/au/autobrr/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  cacert,
+  buildGoModule,
+  fetchFromGitHub,
+  stdenvNoCC,
+  nix-update-script,
+  nodejs,
+  nodePackages,
+  pnpm_9,
+  typescript,
+}:
+
+let
+  pname = "autobrr";
+  version = "1.52.0";
+  src = fetchFromGitHub {
+    owner = "autobrr";
+    repo = "autobrr";
+    rev = "v${version}";
+    hash = "sha256-H9XgRcKgs2YgeedXg5tyZYa3EHNeC/Xa2uC6OPqoX+8=";
+  };
+
+  autobrr-web = stdenvNoCC.mkDerivation {
+    pname = "${pname}-web";
+    inherit src version;
+
+    nativeBuildInputs = [
+      nodejs
+      pnpm_9.configHook
+      typescript
+    ];
+
+    sourceRoot = "${src.name}/web";
+
+    pnpmDeps = pnpm_9.fetchDeps {
+      inherit (autobrr-web)
+        pname
+        version
+        src
+        sourceRoot
+        ;
+      hash = "sha256-TcfxE18nkOSxa4UG2euK1hViPw8tY0TS6tayGJsNNog=";
+    };
+
+    postBuild = ''
+      pnpm run build
+    '';
+
+    installPhase = ''
+      cp -r dist $out
+    '';
+  };
+in
+buildGoModule rec {
+  inherit
+    autobrr-web
+    pname
+    version
+    src
+    ;
+
+  vendorHash = "sha256-Oq+g4AjjdNHY7hkFtgGkr899RR0eGw9NxqsiFHKNQEg=";
+
+  preBuild = ''
+    cp -r ${autobrr-web}/* web/dist
+  '';
+
+  ldflags = [
+    "-X main.version=${version}"
+    "-X main.commit=${src.rev}"
+  ];
+
+  passthru.updateScript = nix-update-script { };
+
+  meta = {
+    description = "Modern, easy to use download automation for torrents and usenet";
+    license = lib.licenses.gpl2Plus;
+    homepage = "https://autobrr.com/";
+    maintainers = with lib.maintainers; [ av-gal ];
+    mainProgram = "autobrr";
+  };
```

Comment:
I'm not sure what to put here. Do I match what is built upstream, or set `lib.platforms.all` in the hopes that it will work on more exotic OSs as well?